doc/optipng-0.7.8-win64/doc/authors.txt

```diff
@@ -0,0 +1,31 @@
+# OptiPNG version 0.7.8
```

Comment:
I will look more fully next week. But in the meantime, these files should not be added